tests/std/tests/P0009R18_mdspan_extents_death/test.cpp

```diff
@@ -54,6 +54,13 @@ void test_construction_from_pack_with_unrepresentable_as_index_type_values_2() {
     [[maybe_unused]] extents<unsigned char, 1, dynamic_extent> e{ConvertibleToInt<unsigned char>{.val = 1}, 256};
 }
 
+void test_construction_from_pack_with_unrepresentable_as_index_type_values_3() {
+    static_assert(signed_integral<char>, "char is not signed integral");

```

Comment:
I find this `static_assert` message to be confusing, because it's somewhat ambiguous whether it's mentioning something that should be the case, or something that actually is the case. Here's it's "actually is the case", but there's still some residual confusion because it's not talking about any actual Standardese guarantee, just an implementation/test assumption.

I think it would be better to replace this message with `"This test assumes that it isn't being compiled with /J"` or something along those lines.